install/kubernetes/helm/istio/values.yaml

```diff
@@ -310,6 +310,7 @@ pilot:
   enabled: true
   replicaCount: 1
   image: pilot
+  traceSampling: 100.0
```

Comment:
Doesn't seem the right default - who would run with 100% sampling ? Ok for the demo.

Reply:
(demo uses overrides - the defaults should match what we expect a normal user will have for non-demo )